drivers/counter/timer_tmr_cmsdk_apb.c

```diff
@@ -87,36 +89,86 @@ static u32_t timer_tmr_cmsdk_apb_read(struct device *dev)
 	return value;
 }
 
+static int timer_tmr_cmsdk_apb_set_wrap_alarm(struct device *dev,
+					      u32_t ticks,
+					      counter_wrap_callback_t callback,
+					      void *user_data)
+{
+	const struct timer_tmr_cmsdk_apb_cfg * const cfg =
+						dev->config->config_info;
+	struct timer_tmr_cmsdk_apb_dev_data *data = dev->driver_data;
+
+	/* Set callback */
+	data->user_cb = callback;
+
+	/* Store the reload value */
+	data->load = ticks;
+
+	/* Set value register to count */
+	cfg->timer->value = ticks;
+
+	/* Set the timer reload to count */
+	cfg->timer->reload = ticks;
+
+	/* Enable IRQ */
+	cfg->timer->ctrl |= TIMER_CTRL_IRQ_EN;
+
+	return 0;
+}
+
+static u32_t timer_tmr_cmsdk_apb_get_wrap(struct device *dev)
+{
+	struct timer_tmr_cmsdk_apb_dev_data *data = dev->driver_data;
+
+	u32_t ticks = data->load;
+
+	return ticks;
+}
+
 static int timer_tmr_cmsdk_apb_set_alarm(struct device *dev,
```

Comment:
similar thing as in other driver, it seems like `set_wrap` should not be implemented. only `set_wrap`